validator/gke_disable_default_service_account_test.rego

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPGKEDisableDefaultServiceAccountConstraintV1
+
+import data.validator.gcp.lib as lib
+
+all_violations[violation] {
+	resource := data.test.fixtures.gke_disable_default_service_account.assets[_]
+	constraint := data.test.fixtures.gke_disable_default_service_account.constraints.disable_gke_default_service_account
+
+	issues := deny with input.asset as resource
+		 with input.constraint as constraint
+
+	violation := issues[_]
+}
+
+test_nodepool_limited_service_account {
```

Comment:
Could we scale a bit these tests? I would like to check that the assets in violation are exactly equal to a set of asset names that should violate in our data set.
You can check [this pr](https://github.com/forseti-security/policy-library/blob/master/validator/allowed_resource_types_test.rego#L59) for instance